src/ConfigDefinition.php

```diff
@@ -11,6 +11,11 @@ class ConfigDefinition extends BaseConfigDefinition
 {
     private function isValidEncoding(string $encoding) : bool
     {
+        $mbEncodings = ['UCS-2', 'UCS-2LE', 'UCS-4', 'CSUCS4', 'UCS-4BE', 'UCS-4LE', 'UTF-16',
+            'UTF-16BE', 'UTF-16LE', 'UTF-32', 'UTF-32BE', 'UTF-32LE', 'UTF-7'];
+        if (in_array(strtoupper($encoding), $mbEncodings)) {
+            return true;
```

Comment:
proč je tady tahle zkratka? `iconv($encoding, 'UTF-8', 'abc');` nefunguje?

Reply:
nefunguje, protoze kdyz je to utf-16 nebo ucs-cokoliv (cokoliv nekompatibilniho s ascii), tak to skonci chybou 'invalid multibyte string', protoze v `abc` je v utf16 hex `61 00 62 00 63 00`, ale v utf-8 hex `61 62 63`. Tj. pokud tam posleme hex `61 62 63`, tak to neni validni utf-16/ucs a iconv failne

Reply:
jasně! tak potom bych byl pro, aby se mb/nonmb encodingy testovaly stejně, přidat je do toho pole.

Reply:
jakoze tam strcit tohle https://gist.github.com/hakre/4188459 ?

Reply:
Každopdáně, pokud tam zlstane tohle, tak prosím komentář do kódu, protože je to fakt nepochopitelné.

Reply:
@odinuv ideálně `iconv -l` přímo z docker image

Reply:
aha! ono totiz na alpine nefunguje

![image](https://user-images.githubusercontent.com/4319320/40655606-2de37a08-6342-11e8-8c5e-7004a592d2fb.png)

ale na php7 to funguje!, jdu to predelat